presto-main/src/test/java/com/facebook/presto/tdigest/TestTDigest.java

```diff
@@ -52,6 +54,7 @@
 {
     private static final int NUMBER_OF_ENTRIES = 1_000_000;
     private static final int STANDARD_COMPRESSION_FACTOR = 100;
+    private static final double MAX_COMPRESSION_FACTOR = 1_000;
```

Comment:
Maybe change the visibility to package private (just remove `private` and don't specify a visibility modifier) and static import directly from the original type?